dae/dae/configuration/tests/conftest.py

```diff
@@ -1,7 +1,8 @@
 import os
 import pytest
 
-from dae.configuration.gpf_config_parser import validate_path
+from dae.configuration.utils import \
+    validate_path
```

Comment:
Why have you split this line? It is shorter than 80.

Reply:
fixed